nvidia-oot/6.0/0001-Porting-nvidia-driver-patches-to-jetpack-6.0.patch

```diff
@@ -1224,6 +1224,15 @@ diff --git a/include/media/tegra_camera_core.h b/include/media/tegra_camera_core
 index e6d9a2b1..38634b77 100644
 --- a/include/media/tegra_camera_core.h
 +++ b/include/media/tegra_camera_core.h
+@@ -18,7 +18,7 @@
+ /* Width alignment */
+ #define TEGRA_WIDTH_ALIGNMENT	1
+ /* Stride alignment */
+-#define TEGRA_STRIDE_ALIGNMENT	1
++#define TEGRA_STRIDE_ALIGNMENT	64
```

Comment:
Reference from JP 5.0.2 patch:
https://github.com/IntelRealSense/realsense_mipi_platform_driver/blob/ddcafa591191c2c29fc235d95fc8e3071cbd8c91/kernel/nvidia/5.0.2/0001-Porting-driver-patches-to-jetpack-5.0.2.patch#L2084